src/components/app/MenuButtons.js

```diff
@@ -589,6 +604,69 @@ const MenuButtons = ({
   predictUrl,
 }: MenuButtonsProps) => (
   <div className="menuButtons">
+    <ButtonWithPanel
+      className="menuButtonsOpenMetainfo"
+      label="Profile info"
+      panel={
+        <ArrowPanel className="arrowPanelOpenMetainfo">
+          <h2 className="arrowPanelSubTitle">Timing</h2>
+          <div className="arrowPanelSection">
+            <span>
+              <span className="metainfolabel">startTime: </span>
+              {_formatDate(profile.meta.startTime)}
```

Comment:
I think it would be easier to achieve consistent styling if the value is also wrapped inside of an element, and is not just a plain text sibling of the label which is wrapped in an element. Then you can use grid layout just like in the sidebar or in the tooltip.